README.md

```diff
@@ -1 +1,129 @@
-# Portfolio-finish-mobile-version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: finish mobile version 
+
+> This is the finsh version of the Porfolio template. The design are given by the 
+> Microverse that I have made it 
+
+
+## Built With
+
+- HTML, CSS
+- Linters for checking error
+- Technologies used
+
+
+
+## Getting Started
+
+This project incilude card and About section 
+In other to run this you need a local copy. Fist you need to setup a Linters adn the follow the coomad and the end I will give a 
+screenshot that there is no error in linters 
+

```

Comment:
- [ ] [optional] This getting started is not descriptive enough. Kindly illustrate how your repo can be accessed and cloned to a local machine. 😄